app/static/js/courseManagement.js

```diff
@@ -0,0 +1,19 @@
+function changeTerm() {
+  var termId = $('option:selected', selectTerm).attr('value');
+  console.log("lalalalalalal", termId);
+  currentRequest = $.ajax({
+    type: "POST",
+    url: "/courseManagement" + "/" ,
+    data: JSON.stringify(termId),
+    contentType: "application/json",
+    success: function(response){
+
+      }
+    })
+    $('form').submit();
+};

```

Comment:
I'm confused about this ajax call that does nothing on success, and this form submission that doesn't wait for the ajax to finish